hazelcast-jet-core/pom.xml

```diff
@@ -132,5 +133,10 @@
             <version>2.4.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>${h2.version}</version>
```

Comment:
should have test scope, you can also embed the version like with the other test libraries